nighthawk/source/client/benchmark_client_impl.h

```diff
@@ -43,13 +43,20 @@ struct BenchmarkClientStats {
   ALL_BENCHMARK_CLIENT_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+class PrefetchablePool {
```

Comment:
Does this belong as a pure `include` interface?

Reply:
Yes, let me add that.